pylint/checkers/variables.py

```diff
@@ -1392,8 +1392,8 @@ def _in_lambda_or_comprehension_body(
     def _is_variable_violation(
```

Comment:
I think the submodule were added by mistake, but I wondered if we should add them for real. It's probably a waste of time for runner that do not launch the primer.

Reply:
Mistake indeed. Not sure why that happened. Don't think we should add them, at least not in this PR.

Reply:
The proper `.gitignore` for `.pylint_primer_test` is only in the primer test branch right now, so after getting them a `git add .` in another branch will stage them.